pages/reset-pass.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from "@/components/shared/buttons/Button/Button";
+import Input from "@/components/shared/forms/Input/Input";
+import Layout from "@/components/shared/layout/Layout";
+import Heading from "@/components/shared/typography/Heading/Heading";
+import Link from "next/dist/client/link";
```

Comment:
It's interesting solution, I've never known this link can imported like that. Whatever, the next/link docs recommends import one from `next/link` library. Check the link https://nextjs.org/docs/pages/api-reference/components/link
```suggestion
import Link from "next/link";
```